tests/test_mod.py

```diff
@@ -502,6 +502,42 @@ def test_get_product_odata_short():
             assert ret[k] == expected[k]
 
 
+def scrub_string(string, replacement=''):
+    '''Scrub a string from a VCR response body string
+    '''
+    def before_record_response(response):
+        response['body']['string'] = response['body']['string'].replace(string, replacement)
+        return response
+    return before_record_response
+
+@pytest.mark.scihub
+def test_get_product_odata_short_with_missing_online_key():
+    api = SentinelAPI(**_api_auth)
+
+    uuid = '8df46c9e-a20c-43db-a19a-4240c2ed3b8b'
+    expected_short = {
+        'id': '8df46c9e-a20c-43db-a19a-4240c2ed3b8b',
+        'size': 143549851,
+        'md5': 'D5E4DF5C38C6E97BF7E7BD540AB21C05',
+        'url': "https://scihub.copernicus.eu/apihub/odata/v1/Products('8df46c9e-a20c-43db-a19a-4240c2ed3b8b')/$value",
+        'date': datetime(2015, 11, 21, 10, 3, 56, 675000),
+        'footprint': 'POLYGON((-63.852531 -5.880887,-67.495872 -5.075419,-67.066071 -3.084356,-63.430576 -3.880541,'
+                     '-63.852531 -5.880887))',
+        'title': 'S1A_EW_GRDM_1SDV_20151121T100356_20151121T100429_008701_00C622_A0EC',
+        'Online': True,
+        'Creation Date': datetime(2015, 11, 21, 13, 22, 1, 652000),
+        'Ingestion Date': datetime(2015, 11, 21, 13, 22, 4, 992000),
+    }
+
+    # scrub 'Online' key from response
+    with my_vcr.use_cassette("test_get_product_odata_short.yaml",
+                             before_record_response=scrub_string(b'"Online":false,', b'')):
```

Comment:
This should be `False` to be consistent with PEP285.

Reply:
IMO PEP285 does not apply here because the the string passed to the `scrub_string` method is not referring to a Python variable but a value inside the [`test_get_product_odata_short.yaml`](https://github.com/viktorbahr/sentinelsat/blob/d3046e8ec385a23f9d04d070adccd0a620f48abc/tests/fixtures/vcr_cassettes/test_get_product_odata_short.yaml#L12) fixture.

Changing to uppercase `b'"Online":False,'` would not match the target value in the respective fixture, wouldn't it?

Reply:
You are absolutely right. My bad - I didn't catch that this was taken verbatim from the fixture.